crypto/asn1/a_int.c

```diff
@@ -251,7 +251,7 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
             ASN1err(ASN1_F_ASN1_GET_INT64, ASN1_R_TOO_SMALL);
             return 0;
         }
-        *pr = (int64_t)(0 - r);
+        *pr = -(int64_t)r;
```

Comment:
@kroeckx pointed out the other day that it's better to use `0 - value`...  apparently, some VS versions give out warnings otherwise.